src/tfrnnlmbin/Makefile

```diff
@@ -35,6 +36,6 @@ ADDLIBS = ../lat/kaldi-lat.a ../lm/kaldi-lm.a ../fstext/kaldi-fstext.a \
 LDLIBS +=  -lz -ldl -fPIC -lrt
 LDLIBS += -L$(TENSORFLOW)/bazel-bin/tensorflow -ltensorflow_cc -ltensorflow_framework
 
-LDFLAGS += -Wl,-rpath=$(shell pwd)/../../tools/tensorflow/bazel-bin/tensorflow/
+LDFLAGS += -Wl,-rpath,$(shell pwd)/../../tools/tensorflow/bazel-bin/tensorflow/
```

Comment:
I think we should also use `$(TENSORFLOW)` here too instead of `$(shell pwd)/../../tools` part. The variable is defined at start of this file to exactly that value.